src/log.cc

```diff
@@ -60,6 +67,7 @@ Message::Message(Verbosity verbosity, const char *file, int line)
     case Verbosity_ERROR: stream_ << 'E'; break;
     case Verbosity_WARNING: stream_ << 'W'; break;
     case Verbosity_INFO: stream_ << 'I'; break;
+    case Verbosity_DEBUG: stream_ << 'D'; break;
```

Comment:
`LOG_V(1)` already does what `LOG_V(DEBUG)` will do..

Reply:
DEBUG is more readable than 1. It's unclear what 1 means.